react-backend/client/src/api/index.js

```diff
@@ -172,6 +172,10 @@ export const Visitors = {
     );
   },
 
+  getCbName: tkn => axios.get(
```

Comment:
Also this is in the wrong place. Methods in these objects are not organised by _who_ uses them, but by _what_ they provide information about. This is providing information _about_ the CB, and therefore should go in `CbAdmin`, under `get` since it's performing a read operation.

If you look in the existing `CbAdmin.get` method, you'll see a call to `/api/cb/details`, which provides an object describing the CB, including the name, which is what you're looking for. Why not just use that method instead of creating a whole new route?

When developing HTTP APIs like ours, we should seek to design them such that they can comprehensively describe the entities that we are modelling in a concise, but flexible set of endpoints. We should not seek to add an additional route for every single use case if it can be avoided. That is a recipe for a vast and un-maintainable API.

Reply:
I looked at that but I needed an admin token to be able to use that route; I could have added the same route again in the router but checking mwIsAuthenticated instead but I thought that might have been bad practice? I'll move the method to be under CbAdmin.